server/container/guice/mailbox-plugin-deleted-messages-vault-guice/src/main/java/org/apache/james/modules/vault/VaultTaskSerializationModule.java

```diff
@@ -65,18 +69,42 @@
         return BlobStoreVaultGarbageCollectionTaskAdditionalInformationDTO.MODULE;
     }
 
+    @Named(Injections.WEBADMIN_DTO)
+    @ProvidesIntoSet
+    public AdditionalInformationDTOModule<? extends TaskExecutionDetails.AdditionalInformation, ? extends  AdditionalInformationDTO> webAdminBlobStoreVaultGarbageCollectionAdditionalInformation() {
+        return BlobStoreVaultGarbageCollectionTaskAdditionalInformationDTO.MODULE;
```

Comment:
No specific DTO ?

Reply:
or you count on a test to break if the dto change for the webadmin and should not change for the task manager ?

Reply:
https://github.com/linagora/james-project/pull/3382#issuecomment-638732094 answers that ;-)